File: python/raindrops/raindrops.py
def convert(number):
    resp = ''
    if number % 3 == 0:
        resp = resp + 'Pling'
    if number % 5 == 0:
        resp = resp + 'Plang'
    if number % 7 == 0:
        resp = resp + 'Plong'
    return resp if resp else str(number)

File: swift/raindrops/Sources/Raindrops/Raindrops.swift
func raindrops(_ number: Int) -> String {
  var out = ""
  if number % 3 == 0 { out += "Pling" }
  if number % 5 == 0 { out += "Plang" }
  if number % 7 == 0 { out += "Plong" }
  if out.count == 0  { out = "\(number)" }
  return out
}
